components/script/dom/webglrenderingcontext.rs

```diff
@@ -2348,6 +2348,45 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
         Int32Value(receiver.recv().unwrap())
     }
 
+    #[allow(unsafe_code)]
+    // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.7
+    unsafe fn GetRenderbufferParameter(
+        &self,
+        _cx: *mut JSContext,
+        target: u32,
+        pname: u32
+    ) -> JSVal {
+        if self.bound_renderbuffer.get().is_none() {
+            self.webgl_error(InvalidOperation);
+            return NullValue();
+        }
```

Comment:
This should probably move after the target and parameter names, to preserve the same order used in https://www.khronos.org/registry/OpenGL-Refpages/es2.0/xhtml/glGetRenderbufferParameteriv.xml.